sound/soc/sof/amd/acp.c

```diff
@@ -410,6 +411,7 @@ static int acp_reset(struct snd_sof_dev *sdev)
 	if (ret < 0)
 		dev_err(sdev->dev, "timeout in releasing reset\n");
 
+	snd_sof_dsp_write(sdev, ACP_DSP_BAR, desc->acp_clkmux_sel, 0x03);
```

Comment:
@reddysujith it is not clear what you're writing here. Does 0x03 mean you are configuring the clock source to internal or external? And where is the default set?

Reply:
Register has default value right (i.e internal clock source to 96Mhz(0x00)).  we changed from default to 0x03 ACLK(i.e external clock source).